support-frontend/app/services/StripeCheckoutSessionService.scala

```diff
@@ -65,16 +71,16 @@ class StripeCheckoutSessionService(
       .withBody(data)
       .execute()
       .attemptT
-      .leftMap(error => {
-        logger.warn(s"Error creating Stripe checkout session: ${error.getMessage}")
-        "Failed to create Stripe checkout session"
-      })
+      .leftMap(error => error.getMessage)
```

Comment:
I had this previously, but wanted to be sure that no detail about the key could make it back to the controller in the error message. Hence logging the actual error message but returning something hard coded - what do you think?